time/time.c

```diff
@@ -116,6 +116,21 @@ int clock_gettime(clockid_t clk_id, struct timespec *tp)
 }
 
 
+int clock_settime(clockid_t clock_id, const struct timespec *tp)
+{
+	if (clock_id != CLOCK_REALTIME) {
+		errno = EINVAL;
+		return -1;
+	}
+
+	time_t time = tp->tv_sec * 1000 * 1000 + tp->tv_nsec / 1000;
+
+	settime(time);
+
+	return 0;
```

Comment:
Although it's returning success every time, this might change in the future
```suggestion
	return SET_ERRNO(settime(time));
```